lr_face/models.py

```diff
@@ -79,11 +85,19 @@ def predict_proba(self, X: List[FacePair]) -> np.ndarray:
         :return np.ndarray
         """
         scores = []
+        rm_pair = []
         cache_dir = EMBEDDINGS_DIR
         for pair in X:
             embedding1 = self.embedding_model.embed(pair.first, cache_dir)
             embedding2 = self.embedding_model.embed(pair.second, cache_dir)
-            score = spatial.distance.cosine(embedding1, embedding2)
+            if embedding1 is None or embedding2 is None:
+                # If face was not detected, score = -1
+                score = -1
+                scores.append([score, 1 - score])
```

Comment:
this line is executed in both the if and the else, and can thus be moved to below